### PR TITLE
chore(deps): add `@ibm/telemetry-js`

### PR DESCRIPTION
**Pull request by francinelucca**
### Description

Adds config file and dependency necessary to start tracking telemetry data via [IBM Telemetry](https://github.com/ibm-telemetry/telemetry-js)

### Changelog

**New**

- Adds @ibm/telemetry-js dependency
- Adds [telemetry.yml](telemetry.yml) config file
    - This file was generated using [IBM Telemetry automated config generator](https://github.com/ibm-telemetry/telemetry-js-config-generator/pull/1)
- Adds readme telemetry disclosure
- Include `telemetry.yml` in package.json files config 
- Adds postinstall script to use IBM telemetry command

### Review
Please look through package.json config file and ensure all necessary modifications have been made so that the "telemetry.yml" config file is included in the release version of this package.

**PLEASE NOTE:** In order for IBM Telemetry to start collecting data for this project, a new build must be published including these changes.

### CODE REVIEW

**Comment by mattrosno** [comment]
Hi @metonym - can you give this PR a review so we can start tracking telemetry data? Thanks!

**Comment by metonym** [comment]
Thanks for the ping. I'll add this to my queue.

File: README.md [hunk 1]
```diff
@@ -257,3 +257,12 @@ Refer to the [Contributing guidelines](CONTRIBUTING.md).
 
 [npm]: https://img.shields.io/npm/v/carbon-components-svelte.svg?color=262626&style=for-the-badge
 [npm-url]: https://npmjs.com/package/carbon-components-svelte
+
+## <picture><source height="20" width="20" media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/ibm-telemetry/telemetry-js/main/docs/images/ibm-telemetry-dark.svg"><source height="20" width="20" media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/ibm-telemetry/telemetry-js/main/docs/images/ibm-telemetry-light.svg"><img height="20" width="20" alt="IBM Telemetry" src="https://raw.githubusercontent.com/ibm-telemetry/telemetry-js/main/docs/images/ibm-telemetry-light.svg"></picture> IBM Telemetry
+
+This package uses IBM Telemetry to collect metrics data. By installing this
```
Comment (metonym) [inline README.md:263]: Can it explicitly be stated that:

> This package uses IBM Telemetry to collect anonymous metrics data in CI environments.

Comment (francinelucca) [reply README.md]: Done!

**Review by metonym — APPROVED** [review]

**Pull request merged**